pkg/bytecode/value.go

```diff
@@ -59,3 +60,38 @@ func (s stringVal) Equals(v value) bool {
 	}
 	return s == s2
 }
+
+type arrayVal struct {
+	Elements []value
+}
+
+func (a arrayVal) Type() *parser.Type {
+	// Revisit this when adding the typeof builtin: https://github.com/evylang/evy/pull/305/files#r1531149977
```

Comment:
What is that link? It only takes me back to this pull request. I assume it is meant to link to the comment, but that should be `https://github.com/evylang/evy/pull/305#discussion_r1531149977` I think, which is what I get when I "Copy Link" on my comment.